schemas/pioneer-core/demographic-survey/demographic-survey.1.schema.json

```diff
@@ -58,6 +58,9 @@
       },
       "type": "object"
     },
+    "exactIncome": {
```

Comment:
Curious why this is camelCase when other fields look snake_cased. It won't matter on the BigQuery end, but it's a bit inconsistent.

Reply:
I think the schema has been updated a little bit haphazardly. We have `zipCode` as a top-level field, some snake_case for certain field values, and some camelCase still for other field values. I think this should suffice for now; we will do better once we're on Glean.